src/tye/Program.RunCommand.cs

```diff
@@ -72,18 +72,21 @@ private static Command CreateRunCommand(string[] args)
                     throw new CommandException("No project or solution file was found.");
                 }
 
-                var output = new OutputContext(console, Verbosity.Quiet);
-                var application = await ApplicationFactory.CreateAsync(output, path);
-
-                await application.ProcessExtensionsAsync(ExtensionContext.OperationKind.LocalRun);
-
-                InitializeThreadPoolSettings(application.Services.Count);
+                var output = new OutputContext(console, Verbosity.Info);
 
+                output.WriteInfoLine("Loading Application Details...");
+                var application = await ApplicationFactory.CreateAsync(output, path);
                 if (application.Services.Count == 0)
                 {
                     throw new CommandException($"No services found in \"{application.Source.Name}\"");
                 }
 
+                await application.ProcessExtensionsAsync(ExtensionContext.OperationKind.LocalRun);
+
+                InitializeThreadPoolSettings(application.Services.Count);
+
+                output.WriteInfoLine("Launching Tye Host...");
+                output.WriteInfoLine(string.Empty);
```

Comment:
Blank line here is because we're changing from one logging format to another. Makes it look better, and makes it clear that the host is taking over.